lib/elixir/src/elixir_parser.yrl

```diff
@@ -857,6 +857,21 @@ warn_empty_stab_clause({stab_op, {Line, _Begin, _End}, '->'}) ->
     "an expression is always required on the right side of ->. "
     "Please provide a value after ->").
 
+%% TODO: Make this an error on Elixir v2.0.
```

Comment:
I think a warning is better than an error in this case, so we can remove the TODO. :D